src/generateSummary.ts

```diff
@@ -0,0 +1,54 @@
+import * as fs from 'fs';
+
+import { getSafeL2SingletonDeployment, getSafeSingletonDeployment } from './safes';
+import { getProxyFactoryDeployment } from './factories';
+import { getDefaultCallbackHandlerDeployment, getCompatibilityFallbackHandlerDeployment, getFallbackHandlerDeployment } from './handler';
+import { getMultiSendDeployment, getMultiSendCallOnlyDeployment, getCreateCallDeployment, getSignMessageLibDeployment } from './libs';
+import ALL_CHAINS from '../chains.json';
+
+const title = '# Safe Deployments\n';
+const description = 'This table contains a list of deployed Safe contracts. Chain information was taken from [Ethereum Lists: Chains](https://github.com/ethereum-lists/chains).\n';
+const versions = ['1.3.0', '1.2.0', '1.1.1', '1.0.0'];
```

Comment:
Maybe it would be better just to load the folder in the `assets` directory. This way the script doesn't need to be adjusted when a new version is added.

Reply:
Done: https://github.com/safe-global/safe-deployments/pull/228/commits/c2ad2b8335e576c9e010968627af6736aedee6d8